client/src/components/Post/index.tsx

```diff
@@ -80,17 +90,17 @@ export default class Post extends React.Component<IProps> {
     }
 
     public onDeleteLike = (): void => {
-        const {_id : userId}: any = this.props.user;
-        const {_id : postId}: any = this.props.userPosts.selectedPost;
+        const {_id: userId}: any = this.props.user;
+        const {_id: postId}: any = this.props.userPosts.selectedPost;
```

Comment:
const {
    userPosts : { selectedPost : {_id: postId }  } ,
    user: {_id: userId},
}  = this.props;